src/WebUI/Utils.ts

```diff
@@ -86,4 +86,11 @@ export class Utils {
         } 
         return Statuses.Failed;
     }
+
+    public static filterByName(objectMeta: V1ObjectMeta, filterKey?:string): boolean {

```

Comment:
as the name of the function saying filterByName, should we send metadata name only in to this function

Reply:
Done. modified the fn signature to accept metadata.name instead of entire metadata